packages/calendar-utils/src/calendar-body/calendar-body.styles.js

```diff
@@ -127,66 +154,17 @@ const getInputContainerStyles = (props, state) => {
       border-color: ${vars.borderColorForInputWhenFocused};
       box-shadow: inset 0 0 0 2px ${vars.borderColorForInputWhenFocused};
     }
+    &:focus,
+    &:hover {
+      border-color: ${props.isDisabled ||
+      props.hasError ||
+      props.hasWarning ||
+      props.isReadOnly ||
+      ((props.isOpen || state.isFocused) && !props.isReadOnly)
+        ? 'inherit'
```

Comment:
This (`inherit`) is what causes the black border that appears when hovering with the calendar open (as seen in https://github.com/commercetools/ui-kit/pull/1670#pullrequestreview-533600128)

Leaving it as an empty string for the truthy condition seems to fix the problem.